### PR TITLE
Add data to data.sql, add queries, and schema, update README

### PR DESCRIPTION
**Pull request by erickhansx**
## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] Created required tables
- [x] Added the required data to tables
- [x] Added the queries to the queries file as required. 
Screenshot 2022-12-15 164331
Screenshot 2022-12-15 202153
Screenshot 2022-12-15 202236
Screenshot 2022-12-15 203218
Screenshot 2022-12-15 210753
Screenshot 2022-12-15 211153
Screenshot 2022-12-15 212400
Screenshot 2022-12-16 080421

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -12,4 +12,69 @@ INSERT INTO animals(name, date_of_birth, escape_attempts, neutered, weight_kg) V
...
+-- What animals belong to Melody Pond?
+SELECT animals.name,owners.full_name FROM animals JOIN owners ON animals.owner_id = owners.id WHERE full_name='Melody Pond';
+-- List of all animals that are pokemon (their type is Pokemon).
+SELECT animals.name,species.name AS type,species.id FROM animals JOIN species ON animals.species_id=species.id AND species.id=1;
+-- List all owners and their animals, remember to include those that don't own any animal.
+SELECT full_name AS name, animals.name AS animal FROM owners LEFT JOIN animals ON owners.id=animals.owner_id;
+-- How many animals are there per species?
+SELECT species.name, COUNT(animals.species_id) FROM species JOIN animals ON species.id=animals.species_id GROUP BY species.name;
+-- List all Digimon owned by Jennifer Orwell.
+SELECT full_name, COUNT(animals.name) FROM owners JOIN animals ON owners.id=animals.owner_id WHERE owners.full_name='Jennifer Orwell' GROUP BY full_name;
```
Comment (MeqdamAlqudah) [inline data.sql]: - [x] Please the query is , list all Digimon so you have to select from animals not owners

File: data.sql [hunk 2]
```diff
@@ -12,4 +12,69 @@ INSERT INTO animals(name, date_of_birth, escape_attempts, neutered, weight_kg) V
...
+-- List all animals owned by Dean Winchester that haven't tried to escape.
+SELECT full_name, COUNT(animals.name) FROM owners JOIN animals ON owners.id=animals.owner_id WHERE owners.full_name='Dean Winchester' AND animals.escape_attempts=0 GROUP BY full_name;
```
Comment (MeqdamAlqudah) [inline data.sql]: - [x] Please add a screenshots of  queries results to the pull request description

File: data.sql [hunk 3]
```diff
@@ -12,4 +12,69 @@ INSERT INTO animals(name, date_of_birth, escape_attempts, neutered, weight_kg) V
...
+-- Who owns the most animals?
+SELECT full_name AS name,COUNT(animals.name) AS animal FROM owners LEFT JOIN animals ON owners.id=animals.owner_id GROUP BY full_name ORDER BY COUNT(*) DESC LIMIT 1;
```
Comment (MeqdamAlqudah) [inline data.sql]: - [x] Please list all queries in the queries.sql file

**Review by MeqdamAlqudah — CHANGES_REQUESTED** [review]
Hi @erickhansx , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Readme file :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Gambit142 — APPROVED** [review]
Hi @erickhansx 👋,

## Status: Approved 🟢   

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights
- All codes for creating, structuring and altering tables are in the _**schema.sql**_ file ✔️ 
- All codes that populate tables with data are in the _**data.sql**_ file ✔️ 
- All codes that query or help to analyze data are in the _**queries.sql**_ file ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**